go.mod

```diff
@@ -20,10 +20,11 @@ replace (
 	github.com/ugorji/go => github.com/ugorji/go v1.1.7
 )
 
-// pinned to grpc v1.26.0
+// pinned to grpc v1.27.0
 replace (
 	github.com/grpc-ecosystem/grpc-gateway => github.com/grpc-ecosystem/grpc-gateway v1.12.2
-	google.golang.org/grpc => github.com/grpc/grpc-go v1.26.0
+	go.etcd.io/etcd => go.etcd.io/etcd v0.5.0-alpha.5.0.20200425165423-262c93980547
```

Comment:
Why are we pinning `etcd` and upgrading `grpc` in this PR?

Reply:
Didn't mean to change etcd, I think I pulled that into my changes when I rebased -  I'll remove that!

grpc needed to be upgraded because nikos has some dependencies which use `grpc.ClientConnInterface` & `grpc.SupportPackageIsVersion6`, which weren't defined in grpc 1.26.0